freqtrade/freqai/data_drawer.py

```diff
@@ -163,37 +164,67 @@ def load_historic_predictions_from_disk(self):
+        exists = self.historic_predictions_folder.exists()
+        convert = self.historic_predictions_path.is_file()
+
         if exists:
             try:
-                with self.historic_predictions_path.open("rb") as fp:
-                    self.historic_predictions = cloudpickle.load(fp)
+                self.load_historic_predictions_from_folder()
                 logger.info(
                     f"Found existing historic predictions at {self.full_path}, but beware "
                     "that statistics may be inaccurate if the bot has been offline for "
                     "an extended period of time."
                 )
             except EOFError:
                 logger.warning(
-                    'Historical prediction file was corrupted. Trying to load backup file.')
-                with self.historic_predictions_bkp_path.open("rb") as fp:
-                    self.historic_predictions = cloudpickle.load(fp)
-                logger.warning('FreqAI successfully loaded the backup historical predictions file.')
+                    'Historical prediction files were corrupted. Trying to load backup files.')
+                self.load_historic_predictions_from_folder()
+                logger.warning('FreqAI successfully loaded the backup '
+                               'historical predictions files.')
+
+        elif not exists and convert:
+            logger.info("Converting your historic predictions pkl to parquet"
+                        "to improve performance.")
+            with Path.open(self.historic_predictions_path, "rb") as fp:
+                self.historic_predictions = cloudpickle.load(fp)
+            self.save_historic_predictions_to_disk()
+            exists = True
 
         else:
-            logger.info("Could not find existing historic_predictions, starting from scratch")
+            logger.warning(
+                f"Follower could not find historic predictions at {self.full_path} "
+                "sending null values back to strategy"
+            )
 
         return exists
 
+    def load_historic_predictions_from_folder(self):
+        """
+        Try to build the historic_predictions dictionary from parquet
+        files in the historic_predictions_folder
+        """
+        for file_path in self.historic_predictions_folder.glob("*.parquet"):
+            key = file_path.stem
+            key.replace("_", "/")
+            self.historic_predictions[key] = pd.read_parquet(file_path)
+
+        return
+
     def save_historic_predictions_to_disk(self):
         """
         Save historic predictions pickle to disk
         """
-        with self.historic_predictions_path.open("wb") as fp:
-            cloudpickle.dump(self.historic_predictions, fp, protocol=cloudpickle.DEFAULT_PROTOCOL)
+
+        self.historic_predictions_folder.mkdir(parents=True, exist_ok=True)
+        for key, value in self.historic_predictions.items():
+            key = key.replace("/", "_")
```

Comment:
I'm not convinced this will work. 
what about futures pairs (on windows)? 

can't name a file `ETH_USDT:USDT` on windows - as `:` is reserved for dir letters.
Best use `pair_to_filename()` from `misc` instead - which will handle this safely.